lib/converters/v2.0.0/converter-v2-to-v21.js

```diff
@@ -0,0 +1,218 @@
+                delete newItem.id;
+            }
+
+            newItem.request = self.request(newItem && newItem.request);
+            newItem.response = _.map(newItem.response || newItem.responses, function (response) {
+                return self.response(response);
+            });
+
+            return newItem;
+        });
+    }
+});
+
+module.exports = {
+    input: '2.0.0',
+    output: '2.1.0',
+    Builders: Builders,
+
+    /**
+     * Converts a single V2 request to a V2.1 request.
+     *
+     * @param {Object} request - The request to convert to v2.1.0.
+     * @param {Object} options - The set of options for the v2.0.0 to v2.1.0 request conversion.
+     * @param {?Function} callback - The function invoked when the conversion has completed.
+     * @returns {*} - The converted v2.1.0 request.
+     */
+    convertSingle: function (request, options, callback) {
+        var err,
+            converted,
+            builders = new Builders(options);
+
+        try { converted = builders.request(request); }
+        catch (e) { err = e; }
+
+        if (callback) { return callback(err, converted); }
+
+        if (err) { throw err; }
+
+        return converted;
+    },
+
+    /**
+     * Converts a single V2 item to a V2.1 request.
+     *
+     * @param {Object} response - The response object to be converted to v2.1.0.
+     * @param {Object} options - The set of options for the current response conversion.
+     * @param {?Function} callback - The function invoked when the response has been converted.
+     * @returns {*}
+     */
+    convertResponse: function (response, options, callback) {
+        var err,
+            converted,
+            builders = new Builders(options);
+
+        try { converted = builders.response(response); }
+        catch (e) { err = e; }
+
+        if (callback) { return callback(err, converted); }
+
+        if (err) { throw err; }
+
+        return converted;
+    },
+
+    /**
+     * Converts a V2 collection to a V2.1 collection (performs ID replacement, etc as necessary).
+     *
+     * @param {Object} collection - The v2.0.0 collection to be converted to v2.1.0.
+     * @param {Object} options - The set of options passed to the current conversion sequence.
+     * @param {?Function} callback - The function invoked when the v2 -> v2.1.0 conversion has completed.
+     * @returns {*}
+     */
+    convert: function (collection, options, callback) {
+        var units = ['item'],
+            builders = new Builders(options),
+            info = collection && collection.info,
+            newCollection = {
+                info: {
+                    _postman_id: builders.options.retainIds ? info && (info._postman_id || info.id) : undefined
+                },
+                // this is done to avoid mutating the original collection
+                variables: (options && options.env) ? builders.variables(options.env) : []
```

Comment:
`variables` key is defined even if none defined?

Reply:
Yes.

Reply:
let's change this for 2.1? any downside to that?